build/Dockerfile

```diff
@@ -20,6 +20,9 @@ FROM golang:1.22-alpine@sha256:8c9183f715b0b4eca05b8b3dbf59766aaedb41ec07477b132
 
 ############################################# Base image for Alpine #############################################
 FROM nginx:1.27.0-alpine@sha256:a45ee5d042aaa9e81e013f97ae40c3dda26fbe98f22b6251acdf28e579560d55 AS alpine
+ARG BUILD_OS=alpine
+
+ENV BUILD_OS=${BUILD_OS}
```

Comment:
I'm not sure these are needed, just setting the `ENV` in `common` might be enough